.scripts/assets-manager/package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "@orchardcore/assets-manager",
+  "version": "1.0.0",
+  "main": "build.js",
+  "type": "module",
+  "bin": "./build.mjs",
+  "dependencies": {
```

Comment:
Update these to the latest: 
![{E665AB70-C89A-4793-AA64-51B43779976B}](https://github.com/user-attachments/assets/06fda96c-eddb-4545-b3b5-1a111cf4e89c)